src/unittests/deskflow/test_ArgParser.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Deskflow -- mouse and keyboard sharing utility
+ * SPDX-FileCopyrightText: (C) 2025 Chris Rizzitello <sithlord48@gmail.com>
+ * SPDX-License-Identifier: GPL-2.0-only WITH LicenseRef-OpenSSL-Exception
+ */
+
+#include "../../lib/deskflow/ArgParser.h"
+#include <QObject>
+
+#include "base/Log.h"
```

Comment:
It's best to order external lib/system includes after our own includes so we don't accidentally solve dependencies. I know `Log.h` isn't Qt at the moment, but the order will confuse others devs if this gets added to.

```suggestion
#include "../../lib/deskflow/ArgParser.h"

#include "base/Log.h"

#include <QObject>
```

Reply:
I has QObject inlcuded first as its the objects base class.